fs.go

```diff
@@ -1153,7 +1170,17 @@ func (h *fsHandler) compressAndOpenFSFile(filePath string, fileEncoding string)
 		return h.newFSFile(f, fileInfo, false, "")
 	}
 
-	compressedFilePath := filePath + h.compressedFileSuffixes[fileEncoding]
+	var compressedFilePath string
+	if h.root == h.compressRoot {
+		compressedFilePath = filePath
+	} else {
+		compressedFilePath = h.compressRoot + path
```

Comment:
Isn't it possible here instead to do something like `if h.root != h.compressRoot { compressedFilePath = strings.Replace(filePath, h.root, h.compressedRoot, 1) }`? That way you don't have to pass the original path in each function.

Reply:
Introduce pathToFilePath and filePathToCompressed https://github.com/mojatter/fasthttp/commit/3dd25f3a3bc0b5693d01d2abcd99b22d88eeefe5

Reply:
I found unnecessary changes, reverted them. https://github.com/valyala/fasthttp/pull/1331/commits/fa6652bf9e2714060e580e92b4954f0673ba15a0